cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,71 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() =>{
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
   });
-});
+
+  it('Assert the pagination and rows count selection', () => {
```

Comment:
i am not sure it is enough to check existing of pagination. Need to assert buttons functionality and also assert table rows count after selecting different values